openapiart/openapiartgo.py

```diff
@@ -1175,11 +1175,14 @@ def _build_setters_getters(self, fluent_new):
                     )
             default = property_schema.get("default")
             if default is not None:
-                if field.type == "number":
-                    default = float(default)
-                if field.type == "bool":
-                    default = str(default).lower()
-                field.default = default
+                if field.setter_method is None:
```

Comment:
if field.type is in self._oapi_go_types.values
then
set default else continue
check if this logic make sense

Reply:
This is good approach. But self._oapi_go_types does not contain all types say ([]string, []int32). I know this is kind of indirect. Otherwise I need to validate any missing type.

Reply:
Use self._oapi_go_types with lstrip